app/views/projects/newfile.html.haml

```diff
@@ -30,7 +30,7 @@
       %div
         - if current_user.id == @project.user.id
           = form_tag file_upload_project_path(@project), multipart: true do
-            = file_field_tag "file" 
+            = file_field_tag "file[]", multiple: true 
             = submit_tag 
```

Comment:
We shouldn't have to make this change in two places - would be great to extract this into a common file. Just leaving this as a note, PR looks good to merge.